charts/kasm-ws/README.md

```
@@ -0,0 +1,2596 @@
+_**Note**: If you want to speicify a namespace, provide the name via the `-n` (or `--namespace`) flag. You need to create the desired namespace beforhand._
+
+The command removes all the Kubernetes components associated with the chart **including persistent volumes** and deletes the release.
+
+## Configuration
+
+Take a look at the [values.yaml](./values.yaml) file of the chart. It contains the default configuration and additional notes.
+So it would be a good idea to take a look at it, even if you already know what you would like to accomplish.
+
+To configure the release (chart values), provide your own values via a *.yaml file and reference it with the `-f` flag.
+
+> #### Sample
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)